MBBSEmu/HostProcess/Structs/UserAccount.cs

```diff
@@ -376,6 +376,28 @@ public byte[] birthd
 
         public const ushort Size = 338;
 
+        /// <summary>
+        ///     Constructor for UserAccount where you can specify the User's UserName and Sex via constructor parameters
+        /// </summary>
+        /// <param name="userName">Username the user will use to log into the system</param>
+        /// <param name="userSex">Sex/Gender of the User (Only M/F supported)</param>
+        public UserAccount(string userName, char userSex = 'M') : this()
```

Comment:
probably best to use an enum for userSex parameter to make it a compile time error rather than a runtime error, but not a huge thing